src/pages/Components/Links/previews/link-icon.preview.html

```diff
@@ -8,7 +8,7 @@
 
   <a
     href="#"
-    class="example__pagination--right mc-link mc-link--primary mc-link--small"
+    class="example__pagination--right mc-link mc-link--s"
```

Comment:
if your div is an example__pagination, it does not feel right the child is a modifier of it : example__pagination--right